ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -366,7 +397,38 @@ module.exports = function CesiumMap(
         },
       })
     },
-    zoomToExtent: function(coords) {},
+    zoomOut: function(opts, done) {},
```

Comment:
Empty function?

Reply:
When I get Cesium's camera to cooperate, this will get an implementation as well. Trying to keep the camera motion from being too jarring.